multimedia/mat2/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                mat2
+version             0.13.3
+categories          multimedia
+license             LGPL-3
+supported_archs     noarch
+platforms           {darwin any}
+maintainers         {@akierig fastmail.de:akierig} \
+                    openmaintainer
+
+description         metadata removal tool
+long_description    mat2 is a metadata removal tool, supporting a wide range \
+                    of commonly used file formats, written in python3.
+homepage            https://0xacab.org/jvoisin/mat2
+master_sites        https://0xacab.org/jvoisin/mat2/-/archive/${version}/
```

Comment:
this should probably just use the default "pypi" as `master_sites` ; just remove this line and make sure to update the `checksums